### PR TITLE
As a user if I am navigating the site and then decide to log in I am redirected to the page I was on prior to logging in.

### PR DESCRIPTION
**Pull request by rshea303**
Added two controllers to manage the path for the traveler show page and host show page.  
Added links on both pages to revert back and forth.  It will need styling and also the links to go back and forth should be moved to a footer or the nav bar.

### CODE REVIEW

**Comment by kyraweber** [comment]
Looks good!

File: app/helpers/travelers_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module TravelersHelper
+end
```
Comment (jmejia) [inline app/helpers/travelers_helper.rb]: @rshea303 If these helpers aren't being used it'd be best to delete them.

File: spec/controllers/hosts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe HostsController, type: :controller do
+
+  describe "GET #show" do
+    it "returns http success" do
+      get :show
+      expect(response).to have_http_status(:success)
```
Comment (jmejia) [inline spec/controllers/hosts_controller_spec.rb]: @rshea303 Nice use of lesser known rspec methods :sparkles:

File: spec/features/admin_item_management_spec.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "Admin Item Management" do
   let(:admin) { create(:admin) }
-  let!(:item) { create(:item) }
+  let!(:listing) { create(:listing) }
```
Comment (jmejia) [inline spec/features/admin_item_management_spec.rb]: @rshea303 Don't think we need `let!` here. It's better to use `let` when possible.

File: spec/features/unauthenticated_user_register_spec.rb [hunk 4]
```diff
@@ -13,7 +13,7 @@
     fill_in "user[password]", with: "password"
     fill_in "user[display_name]", with: "new"
     click_on("Submit")
-    user = User.find(1)
+    user = User.last
```
Comment (jmejia) [inline spec/features/unauthenticated_user_register_spec.rb]: @rshea303 Nice improvement. Much better :+1:

**Pull request merged**